src/rust/ensogl/Cargo.toml

```diff
@@ -43,7 +43,7 @@ smallvec                   = { version = "1.0.0"
 typenum                    = { version = "1.11.2"                                }
 # We require exact version of wasm-bindgen because we do patching final js in our build process,
 # and this is vulnerable to any wasm-bindgen version change.
-wasm-bindgen               = { version = "=0.2.58" , features = ["nightly"]      }
+wasm-bindgen               = { version = "=0.2.60" , features = ["nightly"]      }
```

Comment:
All upgrades of wasm-bindgen version will cause CI to fail, because we do some patches on wasm-bindgen output and apparently those depends on exact version.